ui/src/components/primer/PrimerPage.tsx

```diff
@@ -90,84 +93,66 @@ class PrimerPage extends React.PureComponent<Props> {
     const terms = isEntitiesLoaded(entities) ? glossaryTerms(entities) : [];
     const symbols = isEntitiesLoaded(entities) ? glossarySymbols(entities) : [];
 
-    return ReactDOM.createPortal(
+    const instructions = (
```

Comment:
nit: To me, this is a code smell that `instructions` should instead be a new class of component that is imported into the file.